hedera-node/src/main/java/com/hedera/services/store/AccountStore.java

```diff
@@ -102,12 +132,24 @@ public void persistAccount(Account account) {
 	private void validateUsable(MerkleAccount merkleAccount) {
 		validateTrue(merkleAccount != null, INVALID_ACCOUNT_ID);
 		validateFalse(merkleAccount.isDeleted(), ACCOUNT_DELETED);
+		validateFalse(isAccountExpiredAndPendingRemoval(merkleAccount), ACCOUNT_EXPIRED_AND_PENDING_REMOVAL);
+	}
+
+	private void validateContract(MerkleAccount merkleAccount) {
```

Comment:
Is this used anywhere?

Reply:
sorry I missed it in loadContract method.